sql/src/main/java/io/crate/lucene/LuceneQueryBuilder.java

```diff
@@ -456,11 +459,76 @@ public Query apply(Function input, Context context) throws IOException {
 
         class NotQuery implements FunctionToQuery {
 
+            class SymbolToNotNullContext {
```

Comment:
Could be moved outside of the NotQuery to make it static. But guess it's okay this way as well